ecdsa.go

```diff
@@ -115,28 +102,29 @@ func (m *SigningMethodECDSA) Sign(signingString string, key interface{}) (string
 	hasher := m.Hash.New()
 	hasher.Write([]byte(signingString))
 
-	// Sign the string and return r, s
-	if r, s, err := ecdsa.Sign(rand.Reader, ecdsaKey, hasher.Sum(nil)); err == nil {
-		curveBits := ecdsaKey.Curve.Params().BitSize
-
-		if m.CurveBits != curveBits {
-			return "", ErrInvalidKey
-		}
+	// Sign the string
+	r, s, err := ecdsa.Sign(rand.Reader, ecdsaKey, hasher.Sum(nil))
```

Comment:
Personally, I am not a big fan of changing the `Sign` and `Verify` functions within this PR, even if this is most likely purely cosmetical.

Reply:
Yes, it's just deindentation to make the function more readable, as suggested by go vet / golint / staticcheck.

Reply:
I assume that jwt wishes to follow best practices and the advice of go vet / golint / staticcheck, as long as it does not hinder readability.

Please let me know if this should rather be in a separate pull request.